sonic_package_manager/service_creator/creator.py

```diff
@@ -97,7 +97,7 @@ def remove_if_exists(path):
     log.info(f'removed {path}')
 
 
-def run_command(command: str):
+def run_command(command: List[str]):
```

Comment:
Please be aware type hint does not mean runtime check: https://stackoverflow.com/questions/41692473/does-python-type-hint-annotations-cause-some-run-time-effects. Will you add a runtime check?